### PR TITLE
remove hardcoded values from Portal

### PR DESCRIPTION
**Pull request by cathysarisky**
- [X] There's a clear use-case for this code change, explained below
- [X] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test:all` and `yarn lint`) 

Hi!  First pull request, please forgive me for any rookie mistakes.  I'm learning!

Portal currently has a Globals.js file that should spell out all the colors in use in Portal, which should make it easy to customize the portal colors to match the chosen theme.  Unfortunately, there are a bunch of hardcoded values.  This PR attempts to deal with those.  Perhaps longer-term, portal can take a set of arguments for light vs dark mode, but getting all the values that need to change in a custom Portal build into one file seemed like the place to start!

Most changes will be absolutely invisible.  In a few cases, a hardcoded value was close but not identical to one of the existing color variables, so I chose the closest existing option.

Why isn't the third box checked?  I've got something wrong with my development environment - can't get yarn test:all to run.  I did build it and confirm that it looks fine, no weird color changes or console errors.

### CODE REVIEW

File: ghost/portal/src/components/TriggerButton.styles.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ const TriggerButtonStyles = `
 
     .gh-portal-triggerbtn-label {
         padding: 8px;
-        color: rgb(255, 255, 255);
+        color: var(--white)
```
Comment (peterzimon) [inline ghost/portal/src/components/TriggerButton.styles.js:65]: Missing `;` at EOL

File: ghost/portal/src/components/Frame.styles.js [hunk 2]
```diff
@@ -263,7 +263,7 @@ const FrameStyles = `
 }
 
 .gh-portal-popup-background.preview {
-    background: linear-gradient(45deg, rgba(255,255,255,1) 0%, rgba(249,249,250,1) 100%);
```
Comment (peterzimon) [inline ghost/portal/src/components/Frame.styles.js:266]: This is used in Ghost Admin only, the colors here are not supposed to be changed. Let's keep it hardcoded

File: ghost/portal/src/components/Frame.styles.js [hunk 3]
```diff
@@ -385,9 +385,9 @@ const FrameStyles = `
     align-items: center;
     line-height: 0;
     border-radius: 4px;
-    background: #ffffff;
+    background: var(--white);
```
Comment (peterzimon) [inline ghost/portal/src/components/Frame.styles.js:388]: This logo is marketing material which should be the same everywhere. These values should remain hardcoded.

File: ghost/portal/src/components/Frame.styles.js [hunk 4]
```diff
@@ -385,9 +385,9 @@ const FrameStyles = `
...
     padding: 6px 8px 6px 7px;
-    color: #303336;
+    color: var(--grey1);
```
Comment (peterzimon) [inline ghost/portal/src/components/Frame.styles.js:390]: This logo is marketing material which should be the same everywhere. These values should remain hardcoded.

File: ghost/portal/src/components/Frame.styles.js [hunk 5]
```diff
@@ -399,7 +399,7 @@ const FrameStyles = `
 }
 
 .gh-portal-powered a:hover {
-    color: #15171A;
+    color: var(--grey0);
```
Comment (peterzimon) [inline ghost/portal/src/components/Frame.styles.js:402]: This logo is marketing material which should be the same everywhere. These values should remain hardcoded.

File: ghost/portal/src/components/Frame.styles.js [hunk 6]
```diff
@@ -662,7 +662,7 @@ const FrameStyles = `
     letter-spacing: -.1px;
     font-weight: 700;
     text-transform: uppercase;
-    color: #15212a;
+    color: var(--grey0);
```
Comment (peterzimon) [inline ghost/portal/src/components/Frame.styles.js:665]: This logo is marketing material which should be the same everywhere. These values should remain hardcoded.

File: ghost/portal/src/components/TriggerButton.styles.js [hunk 7]
```diff
@@ -27,7 +27,7 @@ const TriggerButtonStyles = `
         background: var(--brandcolor);
         height: 60px;
         min-width: 60px;
-        box-shadow: rgba(0, 0, 0, 0.24) 0px 8px 16px -2px;
+        box-shadow: rgba(var(--blackrgb), 0.24) 0px 8px 16px -2px;
```
Comment (peterzimon) [inline ghost/portal/src/components/TriggerButton.styles.js:30]: The trigger button shadows and outlines are chosen to work in most situations regardless of the background color. They are not supposed to be changed, let's keep them hardcoded.

**Review by peterzimon — CHANGES_REQUESTED** [review]
Hey @cathysarisky,

Thanks for your PR. Most of your changes are valid however there are a couple of places where hard coded colors are intentional.

1. In general we would not encourage people to change the color of drop shadows in Portal esp. that the same value would/could be used elsewhere. Let's keep the drop shadow colors hard coded to their original values.
2. I left a couple of comments and explanation in specific places which should also remain hard coded.

Thanks again, I'll review and test your PR again once these changes are applied.

Cheers,
Zimo

File: ghost/portal/src/components/TriggerButton.styles.js (hunk 1 above)
Comment (cathysarisky) [reply ghost/portal/src/components/TriggerButton.styles.js:65]: good catch *blush*

**Review by cathysarisky — COMMENTED** [review]
Changes made, thanks @peterzimon !

**Comment by ErisDS** [comment]
Hey @cathysarisky thanks for the updates. @peterzimon is off for a few days and not able to redo the review, but I've added `hacktoberfest-accepted` here in the meantime.

**Comment by cathysarisky** [comment]
> Hey @cathysarisky thanks for the updates. @peterzimon is off for a few days and not able to redo the review, but I've added `hacktoberfest-accepted` here in the meantime.

It's all good!  :)  Thanks.

File: ghost/portal/src/components/common/ProductsSection.js [hunk 8]
```diff
@@ -18,7 +18,7 @@ export const ProductsSectionStyles = ({site}) => {
         .gh-portal-products-pricetoggle {
             position: relative;
             display: flex;
-            background: #F3F3F3;
+            background: var(--grey13);
```
Comment (peterzimon) [inline ghost/portal/src/components/common/ProductsSection.js:21]: There's a subtle but very important difference between the `#F3F3F3` and `var(--grey13)` which makes the contrast worse when using `var(--grey13)`. Let's keep the hard coded value.

File: ghost/portal/src/components/common/ProductsSection.js [hunk 9]
```diff
@@ -60,7 +60,7 @@ export const ProductsSectionStyles = ({site}) => {
             width: 50%;
             color: var(--grey0);
             /*background: white;
-            box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.08);*/
+            box-shadow: 0px 1px 3px rgba(var(--blackrgb), 0.08);*/
```
Comment (peterzimon) [inline ghost/portal/src/components/common/ProductsSection.js:63]: This piece of code is commented, let's just delete it.

**Review by peterzimon — CHANGES_REQUESTED** [review]
Hey @cathysarisky 
Thanks for your changes, as I was testing your code I found two more minor issues. Could you please take a look at those and I'll retest them? 
Thank you, Zimo

**Review by peterzimon — APPROVED** [review]

**Pull request merged**